Source/Core/PlaneGeometry.js

```diff
@@ -0,0 +1,380 @@
+        //>>includeStart('debug', pragmas.debug);
+        Check.typeOf.object('value', value);
+        Check.defined('array', array);
+        //>>includeEnd('debug');
+
+        startingIndex = defaultValue(startingIndex, 0);
+
+        VertexFormat.pack(value._vertexFormat, array, startingIndex);
+
+        return array;
+    };
+
+    var scratchVertexFormat = new VertexFormat();
+    var scratchOptions = {
+        vertexFormat: scratchVertexFormat
+    };
+
+    /**
+     * Retrieves an instance from a packed array.
+     *
+     * @param {Number[]} array The packed array.
+     * @param {Number} [startingIndex=0] The starting index of the element to be unpacked.
+     * @param {PlaneGeometry} [result] The object into which to store the result.
+     * @returns {PlaneGeometry} The modified result parameter or a new PlaneGeometry instance if one was not provided.
+     */
+    PlaneGeometry.unpack = function(array, startingIndex, result) {
+        //>>includeStart('debug', pragmas.debug);
+        Check.defined('array', array);
+        //>>includeEnd('debug');
+
+        startingIndex = defaultValue(startingIndex, 0);
+
+        var vertexFormat = VertexFormat.unpack(array, startingIndex, scratchVertexFormat);
+
+        if (!defined(result)) {
+            return new PlaneGeometry(scratchOptions);
+        }
+
+        result._vertexFormat = VertexFormat.clone(vertexFormat, result._vertexFormat);
+
+        return result;
+    };
+
+    /**
+     * Computes the geometric representation of a plane, including its vertices, indices, and a bounding sphere.
+     *
+     * @param {PlaneGeometry} planeGeometry A description of the plane.
+     * @returns {Geometry|undefined} The computed vertices and indices.
+     */
+    PlaneGeometry.createGeometry = function(planeGeometry) {
+        var vertexFormat = planeGeometry._vertexFormat;
+
+        var min = new Cartesian3(-0.5, -0.5, 0.0);
+        var max = new Cartesian3( 0.5,  0.5, 0.0);
+
+        var attributes = new GeometryAttributes();
+        var indices;
+        var positions;
+
+        if (vertexFormat.position &&
+            (vertexFormat.st || vertexFormat.normal || vertexFormat.tangent || vertexFormat.bitangent)) {
+            if (vertexFormat.position) {
+                // 4 corner points.  Duplicated 3 times each for each incident edge/face.
+                positions = new Float64Array(2 * 4 * 3);
+
+                // +z face
+                positions[0]  = min.x;
+                positions[1]  = min.y;
+                positions[2]  = 0.0;
+                positions[3]  = max.x;
+                positions[4]  = min.y;
+                positions[5]  = 0.0;
+                positions[6]  = max.x;
+                positions[7]  = max.y;
+                positions[8]  = 0.0;
+                positions[9]  = min.x;
+                positions[10] = max.y;
+                positions[11] = 0.0;
+
+                // -z face
+                positions[12] = min.x;
```

Comment:
I don't get why we need to create a front and back face.  We don't do this for `WallGeometry`.  You just need to set the correct attributes for the appearance when you're creating the primitive so it doesn't do backface culling